datasets/submission/submission-guide.qmd

```diff
@@ -17,7 +17,7 @@ We will take you through the process step by step.
 
 We highly recommend that you first do a small submission to our [staging](https://staging-login.bp.nbis.se/) environment, this way we can avoid requiring you to re-upload terabytes of data.
 
-We recommend that you upload two images and generate metadata for them according to the preparation guide, please include all XML files don't forget the `PRIAVET` folder or the `LANDING_PAGE` (in case you want one of those).
+We recommend that you upload two images and generate metadata for them according to the preparation guide, please include all XML files don't forget the `PRIVATE` folder or the `LANDING_PAGE` (in case you want one of those).
```

Comment:
```suggestion
We recommend that you upload two images and generate metadata for them according to the preparation guide. Please include all XML files, and don't forget the `PRIVATE` folder or the `LANDING_PAGE` (in case you want to use either of those).
```

Reply:
I don't quite understand why it is "`PRIVATE` folder **or** the `LANDING_PAGE`", can't it be both of them?

Reply:
What about this

```
We recommend that you upload two images and generate metadata for them according to the preparation guide. Please include all XML files, the `PRIVATE` folder (always required), and the `LANDING_PAGE` folder (if you want a landing page).
```